Translations/pt_BR.po

```diff
@@ -5442,158 +5222,116 @@ msgid "Blackash the Burning"
 msgstr "Blackash, o Calcinante"
 
 #: Source/monstdat.cpp:508
-#, fuzzy
-#| msgid "Shadowcrow"
 msgctxt "monster"
 msgid "Shadowcrow"
 msgstr "Corvo das Sombras"
 
 #: Source/monstdat.cpp:509
-#, fuzzy
-#| msgid "Blightstone the Weak"
 msgctxt "monster"
 msgid "Blightstone the Weak"
-msgstr "Gharbad, o Fraco"
+msgstr "Blightstone, o Fraco"
```

Comment:
"Blight" and "stone" have translations. We can translate this one like the others.